pex/pex_builder.py

```diff
@@ -9,6 +9,7 @@
 import shutil
 
 from pex import pex_warnings
+from pex.bin.sh_boot import create_sh_boot_script
```

Comment:
`pex.pex.builder` -> `pex.bin` is not great. You'd generally want `pex.bin` (a main) dependending on library code but not vice-versa. Perhaps just a follow-up that just moves `sh_boot.py` into the main `pex` package.

Reply:
Sgtm.